Source/UI/Controllers/CardViewController.swift

```diff
@@ -144,6 +145,8 @@ public class CardViewController: UIViewController,
         } else {
             checkoutAPIService?.logger.log(.paymentFormPresented)
         }
+        navigationController?.isNavigationBarHidden = false
```

Comment:
I am not sure you want to play around showing / hiding the navigation bar. Is it possible to change the presentation mode of the VC (ie: before presentation, when you create it, set `.isPresentedModally = true`, which will change behaviour such that this is not an issue?

Reply:
It shouldn't be presented modally, it's pushed to navigation controller without showing bar